### ISSUE TITLE
.env file not loaded if you use the machine ID instead of 'cd'ing to the directory

### ISSUE THREAD

**Issue by d13r** [gosuri/vagrant-env#2]
The .env file is not loaded If you use the machine ID in a command instead of 'cd'ing to the directory - e.g.

``` bash
$ vagrant global-status
id       name    provider   state   directory
----------------------------------------------------------------------------
2173551  default virtualbox running C:/Users/Dave/Documents/Vagrant

$ vagrant suspend 2173551
Error: Required setting 'VM_HOSTNAME' is missing from the .env file.
```

(The error message here is from error checking code I added to make sure all settings were filled in correctly, but it demonstrates that the `.env` file isn't loaded.)

I couldn't find any way to pass a path to `config.env.enable`, and I don't know enough about Vagrant plugin development to determine the path automatically, so my current solution is to bypass the plugin entirely and go straight to Dotenv:

``` ruby
  # config.env.enable
  require 'dotenv'
  Dotenv.load File.dirname(__FILE__) + '/.env'
```

I think this is the same issue as #1, but I don't think it was ever fixed.

Thanks!

**Comment by gosuri** [comment]
Seems there is an issue with Ubuntu. Could you please take a look #4

Will have to revert otherwise

**Comment by gosuri** [comment]
NM. I closed the issue (#4)

**Comment by d13r** [comment]
:+1:
